test/unit/spec/util/index.js

```diff
@@ -31,6 +31,31 @@ describe('Util', () => {
         'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.4 Safari/605.1.15',
         'safari'
       ],
+      [
+        'Safari WKWebView - iPhone',
+        'Mozilla/5.0 (iPhone; CPU iPhone OS 15_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko)',
+        'safari'
+      ],
+      [
+        'Safari WKWebView - iPad',
+        'Mozilla/5.0 (iPad; CPU OS 14_4 like Mac OS X) AppleWebkit/605.1.15 (KHTML, like Gecko) Mobile/15E148',
+        'safari'
+      ],
+      [
+        'Safari iOS - Instagram',
+        'Mozilla/5.0 (iPhone; CPU iPhone OS 15_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 Instagram 220.0.0.8.117 (iPhone12,3; iOS 15_3; en_US; en; scale=3.00; 1125x2436; 347566818) NW/3',
+        'safari'
+      ],
+      [
+        'Safari iOS - Snapchat',
+        'Mozilla/5.0 (iPhone; CPU iPhone OS 15_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.3 Mobile/15E148 Snapchat/11.64.0.38 (like Safari/8612.4.9.0.3, panda)',
+        'safari'
+      ],
+      [
+        'Safari iOS - Slack ',
+        'Mozilla/5.0 (iPhone; CPU iPhone OS 15_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.3 Mobile/15E148 Safari/604.1',
+        'safari'
+      ],
       [
```

Comment:
Is one of these user agents from Tim's app?

Reply:
Yes, the `Safari-WKWebView - iPhone` is from Tim's app.

and `Safari iOS - Slack` is the same as `SFSafariViewController` UA from Tim's app so it seems like most of the applications we tested on is using `SFSafariViewController`.